pkg/costmodel/allocation_helpers_test.go

```diff
@@ -455,7 +455,6 @@ func TestGetUnmountedPodForCluster(t *testing.T) {
 }
 
 func TestCalculateStartAndEnd(t *testing.T) {
-
```

Comment:
can you unit test your change? seems very worth it

Reply:
Yes! Unit tests need to be updated, currently failing.

Because unit tests rely on a mocked Prometheus QueryResult, I'd like to eventually get some integration tests around this to catch regressions.

Reply:
Unit tests now passing.